.travis.yml

```diff
@@ -9,28 +9,37 @@ before_install:
   - gem update --system
   - gem install bundler
 
+env:
+  global:
+    - CC_TEST_REPORTER_ID=4462a28198e28892091f0afb6d13417325549906f26182911786e8ea8735c471
```

Comment:
Should we move this into CI's configuration (in Travis, I can add it or I believe you can too) and remove it from the checked in file?

Reply:
Nice suggestion. Added into Travis. Thanks.